### ISSUE TITLE
Yabai not installable on Tahoe 26.2 M4Pro

### ISSUE THREAD

**Issue by redo7370** [asmvik/yabai#2732]
I tried every trick in the book I could find, but yabai doesn't get added to Accessibility in any way. When trying to start yabai it aborts, because of the missing rights.

**Comment by edward-mb** [comment]
It's a macOS issue that it seems to be fixed in Tahoe 26.3 Beta. I updated to the beta and removed all entries from the `Privacy & Security ~> Accessibility UI`.  Then added the service and restarted `Yabai`. Assuming the --HEAD version is installed, everything seems to work.

Also, there are some workarounds in this repo that don't require installing the Beta and involves removing/updating all entries from TCC and dragging the binaries/symlinks to the Accessibility UI. 

You can see if there are multiple entries for `Yabai` with:
```zsh
sudo sqlite3 "/Library/Application Support/com.apple.TCC/TCC.db" 'SELECT service, client, auth_value, last_modified FROM access;'
```
I think there should be only one `Yabai` entry for `kTCCServiceAccessibility` like:
```sql
kTCCServiceSystemPolicyAllFiles|/opt/homebrew/bin/yabai|2|1767013694
kTCCServiceAccessibility|/opt/homebrew/Cellar/yabai/HEAD-996e26d/bin/yabai|2|1767016724
```

**Comment by wiktor-waj** [comment]
See https://github.com/asmvik/yabai/issues/2688. There is a workaround which is described in the issue I've mentioned. Supposedly this is fixed in beta 26.3
